src/translations/en.json

```diff
@@ -300,8 +300,8 @@
         "by": "by",
         "by_service": "by service",
         "messages": {
-          "was_away": "was away",
-          "was_at_state": "was at {state}",
+          "was_away": "was detected away",
```

Comment:
This one sounds slightly weird, but nicely short. Alternatives:
"was detected being away"
"was detected not being home"